libbeat/common/version.go

```diff
@@ -76,3 +105,16 @@ func (v *Version) LessThan(v1 *Version) bool {
 func (v *Version) String() string {
 	return v.version
 }
+
+func (v *Version) metaIsLessThanOrEqual(v1 *Version) bool {
+	if v.Meta == "" && v1.Meta == "" {
+		return true
+	}
+	if v.Meta == "" {
+		return false
+	}
+	if v1.Meta == "" {
+		return true
+	}
+	return v.Meta <= v1.Meta
```

Comment:
This ofc is not generically valid, but I think with our current version handling (`alpha, beta, rc1, ..`) it should be good enough.

Reply:
nice trick :-) I assume then <= will check the alphabetical order?

Reply:
yes it seems to use alphabetical order